app/scripts/lib/assertion.js

```diff
@@ -10,7 +10,7 @@ define(function (require, exports, module) {
   const requireOnDemand = require('./require-on-demand');
 
   const CERT_DURATION_MS =  new Duration('6h').milliseconds();
-  const ASSERTION_DURATION_MS = new Duration('52w').milliseconds() * 25; //25 years
+  const ASSERTION_DURATION_MS = CERT_DURATION_MS;
```

Comment:
I wonder in addition to this, if we should set the requested assertion duration to be `CERT_DURATION_MS - (amount of time needed for XHR request to sign the certificate)`.